src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftBuilder.java

```diff
@@ -41,7 +43,21 @@ default String getDisplayName() {
 	String getWaitTime();
 	
 	String getWaitTime(Map<String, String> overrides);
-	
+
+	// Used for if a user specifies a time unit with a wait time. Unitless numbers are considered milliseconds.
+	default long convertUnitNotation(String value) {
+		if(value.matches("\\d+milli")) {
+			return Long.valueOf(value.substring(0,value.length()-2));
```

Comment:
Sorry, I missed that from when I changed from ms/s/m to milli/sec/min.  It's fixed now